src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int RANDOM_BOUND = 30;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int numberOfFigures = random.nextInt(FIGURE_COUNT - 1); //because from 0 to 4
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
Move Random and ColorSupplier object outside of methods, make them as class fields. If you do that, objects will created once, not all the time when you will run method